### PR TITLE
Add a script to list all CSS properties parsed by Servo.

### PR DESCRIPTION
**Pull request by SimonSapin**
I’ve been asked for that list by two different people this week :)

r? @larsbergstrom

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9333)

### CODE REVIEW

**Comment by SimonSapin** [comment]
@larsbergstrom could we run this on CI and publish the output somewhere?

**Comment by nox** [comment]
Could that be a mach command?

**Comment by SimonSapin** [comment]
Command that prints the list, or runs the smoke test?

**Comment by nox** [comment]
That prints the list.

**Comment by SimonSapin** [comment]
It didn’t seem worth the bother since I don’t expect people to use it interactively (as opposed to a script that maintains some other data), but it could.

**Comment by larsbergstrom** [comment]
@SimonSapin Seems like something we could add to the doc builder - https://github.com/servo/saltfs/blob/master/buildbot/master/master.cfg#L280-L287

File: components/style/list_properties.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import json
+import operator
+
+style = os.path.dirname(__file__)
+sys.path.insert(0, os.path.join(style, "Mako-0.9.1.zip"))
+from mako.template import Template
+
+template = Template(filename=os.path.join(style, "properties.mako.rs"), input_encoding='utf8')
+template.render()
+properties = dict(
```
Comment (frewsxcv) [inline components/style/list_properties.py]: Not worth changing, but FYI: https://www.python.org/dev/peps/pep-0274/

Comment (SimonSapin) [inline components/style/list_properties.py]: Yes, old habits of writing code that runs on 2.6 :)

**Comment by SimonSapin** [comment]
Oops tidy.

@bors-servo r=larsbergstrom

**Comment by LifeIsStrange** [comment]
https://docs.google.com/spreadsheets/d/tdfNeB1mZP5gkgKpd3TWIAQ/htmlview#gid=555855884
There is currently this :)

**Comment by KiChjang** [comment]
@bors-servo retry
* #9336

**Comment by KiChjang** [comment]
@bors-servo retry
* #9286

**Comment by SimonSapin** [comment]
@LifeIsStrange that spreadsheet has to be maintained manually. The point of this script is that its output is always up to date.

**Comment by LifeIsStrange** [comment]
Oh sorry, I had misunderstood !

**Pull request merged**